src/components/Modal/Timeline.vue

```diff
@@ -19,6 +21,17 @@ const labels = {
   max_end: 'Max. end'
 };
 
+const metaStore = useMetaStore();
+
+function getEstimatedTsFromBlock(network, blockNum) {
+  const networkBlockNum = metaStore.currentBlocks.get(network) || 0;
+  const blockDiff = networkBlockNum - blockNum;
+
+  if (blockDiff > 0) return metaStore.currentTs - METADATA[network].blockTime * blockDiff;
+  if (blockDiff < 0) return metaStore.currentTs + METADATA[network].blockTime * Math.abs(blockDiff);
+  return metaStore.currentTs;
```

Comment:
```suggestion
  return metaStore.currentTs - METADATA[network].blockTime * blockDiff;
```

Should be the same I think.

Reply:
Indeed